lib/active_model/serializer.rb

```diff
@@ -112,10 +112,25 @@ def self.digest_caller_file(caller_line)
       Digest::MD5.hexdigest(serializer_file_contents)
     end
 
+    # TODO(beauby): Cache this.
+    def self.serializer_lookup_chain_for(klass)
```

Comment:
some documentation on how this works (when to use, expected input/output, etc) would be fantabulous